iothub_client/src/iothubtransport_amqp_common.c

```diff
@@ -609,7 +609,7 @@ static void on_device_get_twin_completed_callback(DEVICE_TWIN_UPDATE_TYPE update
     (void)update_type;
 
     // Codes_SRS_IOTHUBTRANSPORT_AMQP_COMMON_09_158: [ If `message` or `context` are NULL, the callback shall do nothing and return. ]
-    if (message == NULL || context == NULL)
```

Comment:
Why did you choose to remove this and move it to the other layer? Can't it just remain here?

Reply:
this allows us to free the user context on line 623